src/Integration.Vsix/Suppression/SuppressionManager.cs

```diff
@@ -95,12 +94,12 @@ private bool ShouldIssueBeSuppressed(Diagnostic diagnostic)
 
             // TODO: ?need to make file path relative to the project file path
             // As a minimum, the project, file and rule id must match
-            var issuesInFile = serverIssueProvider.GetServerIssues(liveIssue.ProjectId, liveIssue.IssueFilePath)
-                    .Where(i => StringComparer.OrdinalIgnoreCase.Equals(liveIssue.Diagnostic.Id, i.RuleKey)); // TODO: rule repository?
+            var issuesInFile = sonarqubeIssueProvider.GetSuppressedIssues(liveIssue.ProjectGuid, liveIssue.IssueFilePath)
+                    .Where(i => StringComparer.OrdinalIgnoreCase.Equals(liveIssue.Diagnostic.Id, i.RuleId)); // TODO: rule repository?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)